MoneySplitter.Models/TransactionStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MoneySplitter.Models
+{
+    public enum TransactionStatus
+    {
+        IN_BEGIN,
```

Comment:
Maybe better rename this class to UserTransactionStatus, because it's describes user role, not transaction status